.github/workflows/executable.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - name: Create tar files
         run: |
           tar -cvf percy-osx.tar percy
+          ./percy --version
```

Comment:
swap linux with macos step, and run the executable as separate `run:` 

we usually check end of the step and skip checking intermediate steps.